Firmata.h

```diff
@@ -111,7 +111,7 @@ class FirmataClass
 
     /* private methods ------------------------------ */
     void strobeBlinkPin(byte pin, int count, int onInterval, int offInterval);
-    friend void FirmataMarshaller::sendValueAsTwo7bitBytes(uint16_t value);
+    friend void FirmataMarshaller::sendValueAsTwo7bitBytes(uint16_t value) const;
```

Comment:
What is the compiler optimization of this use of `const`. I was not familiar with this technique and just looked into it. From what I understand it informs the compiler that no class members within the scope of this method can be altered. I'm just not sure what the advantage of this declaration is.